### ISSUE TITLE
Chronic cholestasis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6176]
**Preferred term label:**
Chronic cholestasis

**Synonyms**
Chronic cholestatic diseases

**Definition (free text, please give PubMed ID)**
Characterized by defective bile acid transport from the liver to the intestine, which is caused by primary damage to the biliary epithelium in most cases.
PMID: 10728800 

**Parent term (use hpo.jax.org/app)**
Cholestasis

**Comment by pnrobinson** [comment]
The HPO generally does not have acute/chronic versions for terms, instead we recommend using the clinical modifiers, e.g., this is for Chronic, https://hpo.jax.org/app/browse/term/HP:0011010
